docs/root/configuration/cluster_manager/cluster_stats.rst

```diff
@@ -152,6 +152,10 @@ Circuit breakers statistics will be rooted at *cluster.<name>.circuit_breakers.<
   rq_pending_open, BoolIndicator, Whether the pending requests circuit breaker is closed (false) or open (true)
   rq_open, BoolIndicator, Whether the requests circuit breaker is closed (false) or open (true)
   rq_retry_open, BoolIndicator, Whether the retry circuit breaker is closed (false) or open (true)
+  remaining_cx, Gauge, Number of available connections until the circuit breaker opens
```

Comment:
maybe use `remaining` instead of `available` in the description? here and for the other stats